internal/docstore/dynamodocstore/dynamo.go

```diff
@@ -279,6 +283,75 @@ func (c *collection) primaryKey(a *driver.Action) [2]interface{} {
 	return keys
 }
 
+func (c *collection) runActionsUnordered(ctx context.Context, actions []*driver.Action, opts *driver.RunActionsOptions) driver.ActionListError {
+	errs := make([]error, len(actions))
+	groups, err := c.splitActionsUnordered(actions)
+	if err != nil {
+		errs[0] = err
```

Comment:
Use the index of the bad action here (see below).

Reply:
Done.